feedly/data.py

```diff
@@ -245,5 +250,35 @@ def get_enterprise_tag(self, key:Union[str, EnterpriseStreamId]) -> 'EnterpriseT
 
         return self._get_category_or_tag(id_, self._enterprise_tags, EnterpriseTag, False)
 
-
-
+    def remove_annotations(self, streamable: Streamable, options: StreamOptions = None):
+        for a in streamable.stream_contents(options):
+            if 'annotations' in a.json:
+                for annotation in a.json['annotations']:
+                    if self['id'] == annotation['author']:
+                        self._client.do_api_request(f"v3/annotations/{quote_plus(annotation['id'])}", method='DELETE')
+
+    def remove_tags(self, streamable: Streamable, options: StreamOptions = None):
```

Comment:
can we call this delete_tag and add a big warning in the comment that this is a destructive, non-reversible operation?